platform/linux-generic/pktio/dpdk.c

```diff
@@ -123,8 +123,8 @@ static unsigned cache_size(uint32_t num)
 static inline uint16_t mbuf_data_off(struct rte_mbuf *mbuf,
 				     odp_packet_hdr_t *pkt_hdr)
 {
-	return (uint64_t)pkt_hdr->buf_hdr.seg[0].data -
-			(uint64_t)mbuf->buf_addr;
+	return (unsigned long)pkt_hdr->buf_hdr.seg[0].data -
+			(unsigned long)mbuf->buf_addr;
 }
```

Comment:
uintptr_t would be a more suitable type here.

Reply:
ack